openspec/changes/add-drift-read-cache/tasks.md

```
@@ -3,40 +3,42 @@
 <!-- Co-authored-by: hy3 (opencode-go) -->
 
 ## 1. Drift database
-- [ ] 1.1 Drift database under `data/cache/`; tables mirror read-projection
+- [x] 1.1 Drift database under `data/cache/`; tables mirror read-projection
        DTOs (not the event-store schema)
-- [ ] 1.2 `drift_dev` codegen wired into `build_runner`
+- [x] 1.2 `drift_dev` codegen wired into `build_runner`
 
 ## 2. Repository cache pattern
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)